<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite parallel="tests" name="Failed suite [Suite]" guice-stage="DEVELOPMENT">
  <listeners>
    <listener class-name="RahulShettyAcademy.data.listenrers"/>
  </listeners>
  <test thread-count="5" parallel="tests" name="Sumit Order(failed)">
    <classes>
      <class name="RahulShettyAcademy.Tests.StandAloneTest">
        <methods>
          <include name="launchApplication"/>
          <include name="tearDown"/>
          <include name="orderHistoryTest"/>
          <include name="FirstTest" invocation-numbers="0"/>
        </methods>
      </class> <!-- RahulShettyAcademy.Tests.StandAloneTest -->
    </classes>
  </test> <!-- Sumit Order(failed) -->
  <test thread-count="5" parallel="tests" name="error validation test(failed)">
    <classes>
      <class name="RahulShettyAcademy.Tests.errorValidation">
        <methods>
          <include name="launchApplication"/>
          <include name="tearDown"/>
          <include name="WrongProduct"/>
          <include name="errorMessagetest"/>
        </methods>
      </class> <!-- RahulShettyAcademy.Tests.errorValidation -->
    </classes>
  </test> <!-- error validation test(failed) -->
</suite> <!-- Failed suite [Suite] -->
